romtools/vector_space/__init__.py

```diff
@@ -185,7 +185,7 @@ def extents(self) -> Tuple[int, int, int]:
         return self.__basis.shape
 
 
-class VectorSpaceFromPOD:
+class VectorSpaceFromPOD(VectorSpace):
```

Comment:
i don't think we should do this. THis is misleading IMO. It smells like inheritance but it is actually a protocol.

Reply:
same for the other change similar